spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -38,32 +43,60 @@
       expect(json_response_skill_group_id).to eq(group.id)
     end
 
-    it "responds with error messages if a skill fails to create" do
-      post :create, format: :json, skill: { name: "" }
-
-      expect(response.status).to eq 422
-      expect(json_response_error_message).to eq(["Name can't be blank"])
+    context "when skill is invalid" do
+      it "returns a 422 status code" do
+        post :create, format: :json, skill: { name: nil }
+        expect(response.status).to eq 422
+      end
+
+      it "responds with error messages if name missing" do
+        post :create, format: :json, skill: {
+                                               name: "",
+                                               group: group.name
+                                            }
+
+        expect(json_response_error_message).to eq(["Name can't be blank"])
+      end
+
+      it "responds with error messages if group missing" do
+        post :create, format: :json, skill: {
```

Comment:
For consistency, the braces on this hash (and the attributes within) should be lined up or single-lined unless we're going to change the above hashes to have the JS style (where closing brace aligns with the beginning of the line from which it was written). I'm actually confused as to why Hound is complaining about the others and not this one.